src/components/Books/CreateNewBook.js

```diff
@@ -4,9 +4,9 @@ import { PropTypes } from 'prop-types';
 const CreateNewBook = (props) => {
   const { submitBookToStoreProps } = props;
 
-  const [inputBook, setInputBook] = useState({
-    title: '',
-  });
+  const [inputBook, setInputBook] = useState({ title: '' });
+
+  const [inputBookCategory, setInputBookCategory] = useState('Category');
```

Comment:
- [ ] Let's add the `author`  as we need to be able to know the author of the book. I can see it is not included as one of the inputs in the [design](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek.md) but it is included in the books added. Please let's fix this as it's a little error from the design.

Reply:
@julie-ify 

Thanks for the feedback. It might be okay to add `author` here and we might post it to the API but how can we get the `author` value from the API since its [response](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) is only `title` and `category`.

Reply:
Hi @Mre55,

So sorry, I just tested the API and it truly does not respond to `author's` property. Please submit your pull request again and the next reviewer will see our chat and will approve your PR since the requested change is not valid. Sorry for the misunderstanding. Thank you.

Reply:
You're welcome 💯